src/sentry/api/event_search.py

```diff
@@ -106,9 +106,10 @@
 # "              # quote literal
 quoted_value = ~r"\"((?:[^\"]|(?<=\\)[\"])*)?\""s
 
-# Format tokens
-date_format          = ~r"\d{4}-\d{2}-\d{2}(T\d{2}:\d{2}:\d{2}(\.\d{1,6})?)?Z?(?=\s|\)|$)"
-alt_date_format      = ~r"\d{4}-\d{2}-\d{2}(T\d{2}:\d{2}:\d{2}(\.\d{1,6})?(\+\d{2}:\d{2})?)?(?=\s|\)|$)"
+# Formats
+date_time_1          = ~r"\d{4}-\d{2}-\d{2}(T\d{2}:\d{2}:\d{2}(\.\d{1,6})?)?Z?(?=\s|\)|$)"
```

Comment:
I would maybe label these with the correct format name instead of `_1` and `_2`. I think one of these is probably ISO8601.

Reply:
That sounds good to me